x/rollapp/keeper/invariants.go

```diff
@@ -238,8 +238,8 @@ func RollappFinalizedStateInvariant(k Keeper) sdk.Invariant {
 					broken = true
 				}
 
-				if stateInfo.Status != commontypes.Status_FINALIZED {
-					msg += fmt.Sprintf("rollapp (%s) have stateInfo at index %d not finalized\n", rollapp.RollappId, i)
+				if stateInfo.Status == commontypes.Status_PENDING {
```

Comment:
@omritoptix I think it misses the purpose of this invariant.
all the states until the finalization index expected to be finalized.
I don't see a scenario where it's not